java/app-encryption/src/main/java/com/godaddy/asherah/appencryption/persistence/DynamoDbMetastoreImpl.java

```diff
@@ -42,13 +42,15 @@
 
   // Table instance can be cached since thread-safe and no state other than description, which we don't use
   private final Table table;
+  private final String regionSuffix;
 
   public static Builder newBuilder() {
     return new Builder();
   }
 
-  DynamoDbMetastoreImpl(final DynamoDB dynamoDbDocumentClient) {
+  DynamoDbMetastoreImpl(final DynamoDB dynamoDbDocumentClient, final String region) {
```

Comment:
new argument should be renamed to avoid confusion, i.e. `regionSuffix` or `suffix` as opposed to `region`